common/network-common/src/main/java/org/apache/spark/network/protocol/Encoders.java

```diff
@@ -89,4 +89,27 @@ public static void encode(ByteBuf buf, String[] strings) {
       return strings;
     }
   }
+
+  /** Integer arrays are encoded with their length followed by integers. */
+  public static class IntArrays {
+    public static int encodedLength(int[] ints) {
+      return 4 * (ints.length + 1);
```

Comment:
nit: seems more readable to write `4 + 4 * ints.length`, which matches the doc better.

Reply:
Thanks, done in 6d7f1f3.